src/classes/LVL_LevelAssignBATCH_TEST.cls

```diff
@@ -629,4 +629,175 @@ private class LVL_LevelAssignBATCH_TEST {
         }
     }
 
+    /*********************************************************************************************************
+    @description
+        Execute Level Assign Batch on Contacts having Source Field value outside of all levels
+    verify:
+        The Level Field is changed to blank.
+        The Previous Level Field is populated with the Level Field as it was before the batch has been run.
+    **********************************************************************************************************/  
+    static testMethod void previousLevelShouldBePopulatedWhenCurrentLevelIsChangedToBlank() {
+        Level__c[] levels = new Level__c[] {
```

Comment:
That is correct, already defined test levels covered all ranges so  could not create a test where the field is outside of levels. Although, I see now, I could have used a break between Gold and Platinum levels. 
On the other side, having two levels created in the test method made test clear of what data is within the limit and what is outside of levels.